### PR TITLE
Lab coats can carry emergency oxygen tanks

### PR DESCRIPTION
**Pull request by Aloraydrel**
No reason why a lab coat couldn't hold a small tank

### CODE REVIEW

**Comment by coiax** [comment]
Surprised they can't already, I thought emergency oxygen tanks were universal suit storage items

**Comment by Shapsy** [comment]
Labcoats continue their reign of dominance over lesser exosuits :+1:

**Comment by Bawhoppen** [comment]
>conflicts

**Comment by Niknakflak** [comment]
:-1:

**Comment by TechnoAlchemisto** [comment]
>>>>>merge conflicts

File: code/modules/clothing/suits/labcoat.dm [hunk 1]
```diff
@@ -5,7 +5,7 @@
...
 	body_parts_covered = CHEST|ARMS
-	allowed = list(/obj/item/device/analyzer,/obj/item/stack/medical,/obj/item/weapon/dnainjector,/obj/item/weapon/reagent_containers/dropper,/obj/item/weapon/reagent_containers/syringe,/obj/item/weapon/reagent_containers/hypospray,/obj/item/device/healthanalyzer,/obj/item/device/flashlight/pen,/obj/item/weapon/reagent_containers/glass/bottle,/obj/item/weapon/reagent_containers/glass/beaker,/obj/item/weapon/reagent_containers/pill,/obj/item/weapon/storage/pill_bottle,/obj/item/weapon/paper,/obj/item/weapon/melee/classic_baton/telescopic)
+	allowed = list(/obj/item/device/analyzer,/obj/item/stack/medical,/obj/item/weapon/dnainjector,/obj/item/weapon/reagent_containers/dropper,/obj/item/weapon/reagent_containers/syringe,/obj/item/weapon/reagent_containers/hypospray,/obj/item/device/healthanalyzer,/obj/item/device/flashlight/pen,/obj/item/weapon/reagent_containers/glass/bottle,/obj/item/weapon/reagent_containers/glass/beaker,/obj/item/weapon/reagent_containers/pill,/obj/item/weapon/storage/pill_bottle,/obj/item/weapon/paper,/obj/item/weapon/melee/classic_baton/telescopic,/obj/item/weapon/tank/internals/emergency_oxygen/)
```
Comment (phil235) [inline code/modules/clothing/suits/labcoat.dm]: `/obj/item/weapon/tank/internals/emergency_oxygen` not `/obj/item/weapon/tank/internals/emergency_oxygen/`

**Comment by Aloraydrel** [comment]
I think I did it

**Comment by coiax** [comment]
The PR appears to be now to add two empty lines to `labcoat.dm`.

Are you SURE you did what you meant to do?

**Comment by Aloraydrel** [comment]
fug im retarded

**Comment by coiax** [comment]
Your PR now appears to change nothing.

**Comment by TechnoAlchemisto** [comment]
:clapping_hand:

**Comment by coiax** [comment]
image

Best PR 2016.

**Comment by coiax** [comment]
Also, I'm only teasing, don't be discouraged, being able to use github and git and make changes puts you above 95% of the playerbase. :+1:

**Comment by PKPenguin321** [comment]
never give up!

**Comment by Aloraydrel** [comment]
Wew

**Comment by TechnoAlchemisto** [comment]
You done it

**Pull request merged**